controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -332,6 +332,9 @@ func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) (re
 		&ocsCephObjectStoreUsers{},
 		&ocsCephObjectStores{},
 		&ocsCephRbdMirrors{},
+		&ocsCephNFS{},
+		&ocsCephNFSService{},
```

Comment:
Two things:
1. Following the conventions from above, this should be `ocsCephNFSes` and `ocsCephNFSServices` (plural).
2. Since OCS creates a service for each CephNFS, why not include the service creation and deletion as part of creating/deleting CephNFSes?